app/src/main/res/values-pl/strings.xml

```diff
@@ -0,0 +1,249 @@
+    <string name="menu_item_title_share_session_json">jako JSON (dla Chaosflix)</string>
+    <string name="share_as_text">Udostępnij jako tekst</string>
+    <string name="share_as_json">Udostępnij jako JSON (dla Chaosflix)</string>
+
+    <!-- AlertDialog -->
+    <string name="OK">OK</string>
+
+    <!-- CertificateErrorFragment -->
+    <string name="certificate_error_message">"
+        Nie udało się nawiązać bezpiecznego połączenia.\n
+        (<xliff:g example="Unknown Error" id="error">%s</xliff:g>)
+    </string>
+    <string name="certificate_error_title">Nie udało się dodanie certyfikatu.</string>
+
+    <!-- Error messages -->
+    <string name="dlg_err_connection_failed">Problem z połączeniem</string>
+    <string name="dlg_err_failed_unknown_host">Nieznany adres:\n<xliff:g example="http://example.com" id="url">%s</xliff:g></string>
+    <string name="dlg_err_failed_wrong_http_credentials">Błędna autentykacja HTTP</string>
+    <string name="dlg_err_failed_timeout">Serwer nie odpowiada (timeout).</string>
+    <string name="dlg_err_failed_connect_failure">Nieznany błąd połączenia</string>
+    <string name="dlg_err_failed_not_found">Błędny URL grafiku</string>
+    <string name="dlg_err_failed_parse_failure">Nie mogłem przetworzyć odpowiedzi</string>
+    <string name="dlg_err_failed_ssl_failure">błąd SSL</string>
+    <string name="dlg_err_failed_http_cleartext_not_permitted">HTTP niedozwolone. Proszę użyć HTTPS.</string>
+    <string name="dlg_err_schedule_data">Błąd danych grafiku</string>
+    <string name="dlg_err_schedule_data_empty">
+        Nic do wyświetlenia. Grafik w wersji \"<xliff:g example="0.99" id="version">%s</xliff:g>\"
+        nie zawiera żadnych sesji.
+    </string>
+    <string name="dlg_err_schedule_data_empty_without_version">"
+        Nic do wyświetlenia. Grafik nie zawiera żadnych sesji.
+    </string>
+    <string name="uptodate">Grafik jest aktualny.</string>
+
+    <!-- Schedule changes dialog -->
+    <string name="schedule_changes_dialog_title">Aktualizacja grafiku</string>
+    <string name="schedule_changes_dialog_updated_to_text">Zaktualizowano grafik do wersji\\u0020</string>
+    <string name="schedule_changes_dialog_changed_new_cancelled_text">. Są zmiany w <xliff:g example="3 wykładach" id="num_changes">%1$s</xliff:g>, <xliff:g example="2 są" id="num_new">%2$s</xliff:g> nowe, <xliff:g example="1 jest" id="num_canceled">%3$s</xliff:g> anulowany.</string>
+    <string name="schedule_changes_dialog_affected_text">Zmiany dotyczą <xliff:g example="2" id="num_marked">%1$d</xliff:g> Twoich ulubionych wykładów.</string>
+    <plurals name="schedule_changes_dialog_number_of_sessions">
+        <item quantity="one"><xliff:g example="1" id="count">%d</xliff:g> wykład</item>
+        <item quantity="other"><xliff:g example="9" id="count">%d</xliff:g> wykłady</item>
+        <item quantity="few">Kilka wykładów</item>
+        <item quantity="many">Wiele wykładów</item>
```

Comment:
As it is part of the sentence `Kilka` and `Wiele` have to be lower case I believe. I will change this.

Reply:
:heavy_check_mark: Done.